salt/files/master.d/f_defaults.conf

```diff
@@ -1590,7 +1590,7 @@ mine_get:
 # the coloring of the messages, these color formatters also include padding as
 # well.  Color LogRecord attributes are only available for console logging.
 #
-{{ get_config('log_fmt_console', "'%(colorlevel)s %(colormsg)s'") }}
+#log_fmt_console "%(colorlevel)s %(colormsg)s"
 {{ get_config('log_fmt_console', "'[%(levelname)-8s] %(message)s'") }}
```

Comment:
Line 1594 looks wrong too. I think 'messages' should be 'colormsg' ?

Reply:
No, default is the log level and message, without colors.

Previously:
* not specifying a config gives 2 comment exemples, one with color format, one without color
* specifying a config would give 2 lines with the specified value, and fail to load config

Now, it always get the color message as comment, and could have along the non-color commented, or whatever is specified as format message.